src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/containerTerminated.groovy

```diff
@@ -0,0 +1,22 @@
+podTemplate(yaml:'''
+spec:
+  containers:
+  - name: jnlp
+    image: jenkins/inbound-agent:4.3-4 # alpine flavor has alpine 3.9 which doesn't have stress-ng
+    securityContext:
+      runAsUser: 0
+      privileged: true
```

Comment:
What will this test do if run in a cluster with `PodSecurityPolicy` enabled?

Reply:
Probably timeout since the pod won't be scheduled.

Reply:
Probably. I guess we can deal with that if and when we come to it.